app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -110,6 +110,18 @@ export default class AMRHeatmapVis extends React.Component {
     this.setState({ nodeHoverInfo: null });
   };
 
+  // mediate the callback function for clicking a row label in the heatmap,
+  // so that clicking an allele returns the proper gene
+  onRowLabelClick = rowLabel => {
```

Comment:
This is interesting! If the user clicks on an allele label, we show them information about the gene.

Is there any additional allele-specific information that we can show them as well? We might at least indicate in the sidebar that we are showing the gene for that allele.

We can handle this in a future PR, but worth thinking about.